app/controllers/statistics_controller.rb

```diff
@@ -12,6 +14,31 @@ def collection
     @all_indexers     = build_user_array(DeedType::PAGE_INDEXED)
   end
 
+  def export_csv
+    header = ['User Login', 'User Name', 'Email', 'Opt-In']
+    owner = User.find 'ushmmarchives'
+    collection_ids = owner.collections.map { |c| c.id }.sort
+    deed_map = Deed.where(:collection_id => collection_ids).group(:user_id, :collection_id).count
```

Comment:
Get all the deeds for this collection (because we want to know which users worked on which collections).
   We don't care what kind of deeds the user had, but want to know how many they had to indicate activity on the collection.

Then create an array with `[user_id, collection_id, deed_count]` from the users who worked on the collection.